src/mscorlib/model.xml

```diff
@@ -12100,5 +12100,10 @@
       <Member Name="NonPortableCast&lt;TFrom,TTo&gt;(System.Span&lt;TFrom&gt;)" />
       <Member Name="NonPortableCast&lt;TFrom,TTo&gt;(System.ReadOnlySpan&lt;TFrom&gt;)" />
     </Type>
+    <Type Name="System.ReadOnlySpanExtensions">
```

Comment:
It may be useful to add this with `Condition="FEATURE_SPAN_OF_T` like the other span apis.